code/modules/projectiles/projectile/beams.dm

```diff
@@ -80,6 +84,13 @@
 	muzzle_type = /obj/effect/projectile/muzzle/disabler
 	impact_type = /obj/effect/projectile/impact/disabler
 
+/obj/item/projectile/beam/disabler_borg
+	name = "disabler beam"
+	icon_state = "omnilaser"
+	damage = 30 
+	damage_type = STAMINA
+	eyeblur = 1

```

Comment:
Why do borg disablers get to make peoples eyes blurry.

Reply:
Why do lethal lasers make eyes blurry?

Reply:
Lethals do eyeblur only because there's no other way to convey intense, temporary pain

Reply:
then why dose practice lasers do eye blur 2?

Reply:
There was, it made the vision go hazy, but some idjit replaced that with blurry